10-Sprint-Mission/css/style.css

```
@@ -0,0 +1,168 @@
+body {
+  
+}
+
+.header {
+  width: 1920px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+}
+
+.header a img {
+  max-height: 100%;
+  margin-left: 360px;
+  margin-right: 360px;
+}
+
+.btn.top {
+  position: relative; /* 상대적인 위치 설정 */
+  top: -70px;
+  display: inline-block;
+  background-color: transparent;
+  padding: 0;
+  border: none;
+  font-size: 1.5rem; 
+  font-weight: bold;
+  
+}
+
+.itemlink {
+  cursor: pointer;
+}
+
+.top_banner {
+  width: 1920px;
+  position: relative;
+  background-color: #F0F8FF;
+  text-align: center;
+  padding: 20px 0;
+}
+
+.top_banner img {
+  max-width: 100%;
+  height: auto;
+}
+
+.main-container {
+  
+  height: 2160px;
+  width: 1920px; 
+  display: flex;
+  flex-direction: column;
+ 
+
+}
+
+h5 {
+  color: rgb(44, 67, 214);
+}
+
+.main-item {
+  display: flex;
+  align-items: center;
+  justify-content: space-around; /* 이미지와 텍스트 같이 */
+  background-color: #f8f8f8; /* 각 섹션의 배경색 */
+  padding: 20px;
+  border-radius: 10px; /* 둥근 모서리 추가 */
+}
+
+.main-item.middle {
+  display: flex;
+  align-items: center;
+  justify-content: space-around; /* 이미지와 텍스트 같이 */
+  background-color: #f8f8f8; /* 각 섹션의 배경색 */
+  padding: 20px;
+  border-radius: 10px; /* 둥근 모서리 추가 */
+}
```

P3:
완전 동일한 속성이 두번 선언될 필요가 있을까요?
개발자의 의도가 궁금해지는 코드는 가독성 측면에서 좋지 않으니 가능하면
왜 이렇게 작성하셨는지 알 수 있도록 작성해주시면 좋겠습니다.

```suggestion
// 차라리 이렇게..?
.main-item,
.main-item.middle {
  display: flex;
  align-items: center;
  justify-content: space-around; /* 이미지와 텍스트 같이 */
  background-color: #f8f8f8; /* 각 섹션의 배경색 */
  padding: 20px;
  border-radius: 10px; /* 둥근 모서리 추가 */
}
```